config/tangram.yml

```diff
@@ -7,29 +7,34 @@ site_dir: dist-tangram
 # This means pages should be added or subtracted manually here if the files change.
 pages:
   - Home: index.md
-  - 'Walkthrough': 'walkthrough.md'
-  - Concept overviews:
-    - 'The Scene File': 'Scene-file.md'
-    - 'Filters': 'Filters-Overview.md'
-    - 'Lights': 'Lights-Overview.md'
-    - 'Cameras': 'Cameras-Overview.md'
-    - 'Materials': 'Materials-Overview.md'
-    - 'Shaders': 'Shaders-Overview.md'
-    - 'Styles': 'Styles-Overview.md'
-  - Technical reference:
-    - 'cameras': 'cameras.md'
-    - 'draw': 'draw.md'
-    - 'filters': 'filters.md'
-    - 'layers': 'layers.md'
-    - 'lights': 'lights.md'
-    - 'materials': 'materials.md'
-    - 'scene': 'scene.md'
-    - 'shaders': 'shaders.md'
-    - 'sources': 'sources.md'
-    - 'styles': 'styles.md'
-    - 'textures': 'textures.md'
-    - 'yaml': 'yaml.md'
-  - 'JavaScript API': 'Javascript-API.md'
+  - Scene File:
+    - Concept overviews:
+      - 'The Scene File': 'Scene-file.md'
+      - 'Data Filters': 'Filters-Overview.md'
+      - 'Styles': 'Styles-Overview.md'
+      - 'Shaders': 'Shaders-Overview.md'
+      - 'Raster Sources': 'Raster-Overview.md'
+      - 'Lights': 'Lights-Overview.md'
+      - 'Cameras': 'Cameras-Overview.md'
+      - 'Materials': 'Materials-Overview.md'
+    - Technical reference:
+      - 'cameras': 'cameras.md'
+      - 'draw': 'draw.md'
+      - 'filters': 'filters.md'
+      - 'layers': 'layers.md'
+      - 'lights': 'lights.md'
+      - 'materials': 'materials.md'
+      - 'scene': 'scene.md'
+      - 'shaders': 'shaders.md'
+      - 'sources': 'sources.md'
+      - 'styles': 'styles.md'
+      - 'textures': 'textures.md'
+      - 'yaml': 'yaml.md'
+  - JavaScript API: 'Javascript-API.md'
+  - Demos & Examples:
+    - 'JS Walkthrough': 'JS-awalkthrough.md'
```

Comment:
typo!

On Fri, Apr 22, 2016 at 4:56 PM, Matt Blair <notifications@github.com>
wrote:

> In config/tangram.yml
> <https://github.com/mapzen/mapzen-docs-generator/pull/130#discussion_r60801898>
> :
>
> > +    - Technical reference:
> > +      - 'cameras': 'cameras.md'
> > +      - 'draw': 'draw.md'
> > +      - 'filters': 'filters.md'
> > +      - 'layers': 'layers.md'
> > +      - 'lights': 'lights.md'
> > +      - 'materials': 'materials.md'
> > +      - 'scene': 'scene.md'
> > +      - 'shaders': 'shaders.md'
> > +      - 'sources': 'sources.md'
> > +      - 'styles': 'styles.md'
> > +      - 'textures': 'textures.md'
> > +      - 'yaml': 'yaml.md'
> > +  - JavaScript API: 'Javascript-API.md'
> > +  - Demos & Examples:
> > +    - 'JS Walkthrough': 'JS-awalkthrough.md'
>
> typo?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mapzen/mapzen-docs-generator/pull/130/files/dd506830c2b03f0b4e1277ff8ef5ee4e993c96b5#r60801898>
>